lms/djangoapps/discussion_api/tests/test_forms.py

```diff
@@ -142,6 +143,10 @@ def test_mutually_exclusive(self, params):
             "The following query parameters are mutually exclusive: topic_id, text_search, following"
         )
 
+    def test_invalid_view_choice(self):
+        self.form_data["view"] = "not_a_valid_choice"
+        self.assert_error("view", "Select a valid choice. not_a_valid_choice is not one of the available choices.")
```

Comment:
It's kind of rigid. For example, if someone makes a change as minor as "is not one of the available choices" -> "is not an available choice", the test will break.

It's not that important. I don't have a better suggestion either.